docs/querying/api.md

```diff
@@ -389,7 +389,7 @@ These are APIs that expose database functionalities for the advanced user. These
 We also expose a gRPC API whose definition can be found [here](https://github.com/prometheus/prometheus/blob/master/prompb/rpc.proto). This is experimental and might change in the future.
 
 ### Snapshot
-Snapshot creates a snapshot of all current data into `snapshots/<datetime>-<rand>` under the TSDB's data directory and returns the directory as response.
+Snapshot creates a snapshot of all current data into `snapshots/<datetime>-<rand>` under the TSDB's data directory and returns the directory as response. We replace the `:` in the `datetime` with `_` to be script friendly.
```

Comment:
We never specify the format of the date time, so there's no need to mention the change.